kaggle/cli.py

```diff
@@ -115,7 +116,7 @@ def parse_competitions(subparsers):
       formatter_class=argparse.RawTextHelpFormatter,
       help=Help.command_competitions_list)
   parser_competitions_list_optional = parser_competitions_list._action_groups.pop(
-  )
+      )
```

Comment:
Are the type hints conflicting with the auto-formatter?

Reply:
I should have fixed the format, sorry. I'm going to switch to `black` since that's what `kagglehub` uses and Rider has built-in support for it.